src/Domain/Creators/Shell.hpp

```diff
@@ -102,9 +104,40 @@ class Shell : public DomainCreator<3> {
     static type lower_bound() { return 1; }
   };
 
-  using options = tmpl::list<InnerRadius, OuterRadius, InitialRefinement,
-                             InitialGridPoints, UseEquiangularMap, AspectRatio,
-                             UseLogarithmicMap, WhichWedges, RadialBlockLayers>;
+  template <typename BoundaryConditionsBase>
+  struct InnerBoundaryCondition {
+    static std::string name() noexcept { return "InnerBoundaryCondition"; }
```

Comment:
I think you meant to name this "InnerBoundary", and the one below "OuterBoundary"